src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -31,11 +44,19 @@
  **/
 public class BusinessesService {
 
-  private List<Listing> allBusinesses;
   private final String KEY = "REDACTED";
   private final static Logger LOGGER = 
         Logger.getLogger(BusinessesService.class.getName());
   private final int ALLOWED_SEARCH_REQUESTS = 3;
+  private final int MINFOLLOWERS = 50000;
+  private final int SMALL_BUSINESSES_DISPLAYED = 15;
+  private final String START_SUBSTRING = "| ";
+  private final String END_SUBSTRING = "followers";
+  private final int MIN_NUMBER_OF_MATCHING_BUSINESSES = 5;
```

Comment:
Would "ALLOWED_NUMBER_OF_MATCHING_BUSINESSES" be better here, or "MAX_..." based on what's happening this is bing used as an upper bound to check against.

Reply:
I decided to use ALLOWED_NUMBER_OF_MATCHING_BUSINESSES